### ISSUE TITLE
Production environment thinks it is in Development environment.

### ISSUE THREAD

**Issue by josephdburdick** [sysgears/apollo-fullstack-starter-kit#56]
Hello, I've had some trouble running my project in a production environment (AWS Ubuntu Linux). After a bit of tinkering I decided to git clone the latest version of this repo and run it with NODE_ENV variables set to see what it does and here are the results: 
(Following instructions found in the latest README to install, migrate, seed, etc...)
After running `NODE_ENV=production npm install` ...
<img width="625" alt="screen shot 2017-03-01 at 12 48 02 am" src="https://cloud.githubusercontent.com/assets/1349077/23445338/839124da-fe19-11e6-838c-888c88a464b5.png">
App seems to install with only a few prompts, no big deal. Moving forward...

<img width="823" alt="screen shot 2017-03-01 at 12 48 21 am" src="https://cloud.githubusercontent.com/assets/1349077/23445500/a394f6e8-fe1a-11e6-934e-682dd619aa82.png">

Running within the browser even with the NODE_ENV variable set makes the app think it's still in the dev environment, looking for websockets, etc.

Thoughts? Thanks guys!

**Comment by vlasenko** [comment]
A fresh new clone from this repo works as expected for me:
```
$ git clone https://github.com/sysgears/apollo-fullstack-starter-kit.git
$ npm i
$ NODE_ENV=production npm i
$ NODE_ENV=production npm start
```

Please note that websockets are used in production environment as well for Apollo GraphQL Subscriptions, see file: `src/server/api_server.js`

The code below starts websockets listener for GraphQL subscriptions:
```
new SubscriptionServer({
  subscriptionManager
}, {
  server: server,
  path: '/'
});
```

**Comment by josephdburdick** [comment]
Ok, your previous comment's instructions worked for me on my local machine but not on AWS. But thanks to your comment I tried a few things and ultimately was able to actually build on AWS Ubuntu. Here are the commands I ran to achieve that:
```
npm run clean
npm i
NODE_ENV=production npm i;
npm run build
NODE_ENV=production npm start
```
I would highly recommend that instructions to deploy production be added, without your `npm i; NODE_ENV=production npm i` advice I would have never got this working.
